service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ActionCaseManagementOrderControllerTest.java

```diff
@@ -185,18 +191,12 @@ void aboutToSubmitShouldErrorIfHearingDateInFutureWhenSendToAllParties() throws
     }
 
     @Test
-    void aboutToSubmitShouldReturnCaseManagementOrderWithDraftDocumentWhenNotSendToAllParties() throws Exception {
-        Map<String, Object> data = ImmutableMap.of(
-            CASE_MANAGEMENT_ORDER_JUDICIARY.getKey(), getCaseManagementOrder(),
-            ORDER_ACTION.getKey(), getOrderAction(JUDGE_REQUESTED_CHANGE));
-
-        AboutToStartOrSubmitCallbackResponse response =
-            makeRequest(buildCallbackRequest(data), "about-to-submit");
```

Comment:
Didn't we lose this test case? it's not exactly the same is it?

Reply:
i've added this test case back in as it passes